### DISCUSSION TITLE
Minecraft: Bedrock

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#4] [General]
# Minecraft

A community-driven list of apps and games that work, are emulated or don't work on Windows on ARM

https://windowsonarm.org/

**Comment by lnkd61** [comment]
Minecraft (Bedrock) is currently not ARM native, it is running under x64 emulation for Snapdragon X.

Mojang has a ARM32 binary, but ARM32 support has been deprecated and so the Windows Store will download the x64 binary instead.

**Comment by OpenSource03** [comment]
Thanks, we've renamed Minecraft to Minecraft: Bedrock and also added Minecraft: Java edition as a separate app. Additionally, we've provided community download link for the Java Edition and added the appropriate description for both :)
